app/webpacker/scripts/cookie_banner.spec.js

```diff
@@ -1,36 +1,21 @@
 import CookieBanner from './cookie_banner'
 
-// Test Helpers
-const setCookie = (cookieName, value) => {
-  document.cookie = `${cookieName}=${value};`
-}
-
-const clearCookie = (cookieName) => {
-  document.cookie = `${cookieName}=; expires=Thu, 01 Jan 1970 00:00:00 GMT`
-}
+const cookieName = '_consented_to_apply_cookies'
```

Comment:
Should this be called this? We're not Apply 😆

Reply:
`_consented_to_analytics_cookies`?

Reply:
If you didn't know about Apply (a very generic term btw), you could easily read it as verbatam and it would make sense. But `_consented_to_analytics_cookies` is even clearer. :-)